watson_developer_cloud/watson_developer_cloud_service.py

```diff
@@ -190,9 +190,10 @@ def _get_error_message(response):
             if 'description' in error_json:
                 error_message += ', Description: ' + error_json['description']
             error_message += ', Code: ' + str(response.status_code)
+            return error_message
         except:
-            pass
-        return error_message
+            return {'Error: ': response.text or error_message, 'Code': str(response.status_code)}
```

Comment:
The field name should probably be `'error'` not `'Error: '` (and `'Code'` -> `'code'`)